cmake/onnxruntime_unittests.cmake

```diff
@@ -682,13 +682,19 @@ if (onnxruntime_BUILD_WEBASSEMBLY)
   endif()
 endif()
 
+set(test_all_args)
+if (onnxruntime_USE_TENSORRT)
+    list(APPEND test_all_args "--gtest_filter=-*cpu__*:*cuda__*" )
```

Comment:
comment on what we want to exclude here would be helpful.

Reply:
I added comments here.

Reply:
I don't know where the "cpu__" and "cuda__" in the test names come from - I didn't see them in the code. Explaining that would be helpful.

Reply:
Made the comment more clear.

    # The test names of model tests were using sequential number in the past.
    # This PR https://github.com/microsoft/onnxruntime/pull/10220 (Please see ExpandModelName function in model_tests.cc for more details) 
    # made test name contain the "ep" and "model path" information, so we can easily filter the tests using cuda ep or other ep with *cpu__* or *xxx__*.